data/caasp/update.sh

```diff
@@ -123,12 +143,56 @@ elif [ ! -z "${INSTALL:-}" ]; then
         fi
     done < qam_packages.txt
 
+    if [ ${reboot_required} -eq 1 ]; then
+        # Disable the update repo, so to check if this package is old or new
+        $runner "zypper mr -d UPDATE"
+        new_package=0
+        while read pkg; do
+            if ! zypper se $pkg; then
+                echo "$pkg " >> new_package.txt
+                new_package=1
+            fi
+        done < not_installed.txt
+
+        if [ ${new_package} -eq 1 ]; then
```

Comment:
Maybe use `if [ -f new_package.txt ];` instead of extra `${new_package}` variable - as you did above. Variable seems redundant.